clickstream/src/main/java/software/aws/solution/clickstream/client/ClickstreamConfiguration.java

```diff
@@ -184,22 +184,22 @@ public ClickstreamConfiguration withCompressEvents(final boolean compressEvents)
     }
 
     /**
-     * Is track app lifecycle events.
+     * Is track app screen view events.
      *
      * @return Is track appLifecycle events.
      */
-    public boolean isTrackAppLifecycleEvents() {
-        return this.isTrackAppLifecycleEvents;
+    public boolean isTrackScreenViewEvents() {
+        return this.isTrackScreenViewEvents;
     }
 
     /**
-     * Is track app lifecycle events.
+     * Is track app screen view events.
      *
-     * @param isTrackAppLifecycleEvents Is track app lifecycle events.
+     * @param isTrackScreenViewEvents Is track screen view events.
      * @return the current ClickstreamConfiguration instance.
      */
-    public ClickstreamConfiguration withTrackAppLifecycleEvents(final boolean isTrackAppLifecycleEvents) {
```

Comment:
Is it an API breaking change? If so, please increase the minor version when releasing this change and add a note in release notes.

Reply:
Yes, we will release the changes in version 0.6.0